syft/frameworks/torch/fl/utils.py

```diff
@@ -29,60 +29,67 @@ def extract_batches_per_worker(federated_train_loader: sy.FederatedDataLoader):
     return batches
 
 
-def add_model(dst_model, src_model):
+def add_model(dst_model, src_model,dst_num_data,src_num_data):
     """Add the parameters of two models.
-
         Args:
+
             dst_model (torch.nn.Module): the model to which the src_model will be added.
             src_model (torch.nn.Module): the model to be added to dst_model.
+            dst_num_data: the number of data within dst_model.
+            src_num_data: the number of data within src_model.
```

Comment:
Add typing info
I'd suggest changing the name `dst_num_data` is not very clear neither `the number of data` -> number of data items? size of the dataset?

Reply:
I change it to data_amount